src/Lean/Elab/Tactic/BVDecide/Frontend/Normalize/AC.lean

```diff
@@ -1,39 +1,362 @@
 /-
-Copyright (c) 2025 Lean FRO, LLC. All rights reserved.
+Copyright (c) 2024 Lean FRO, LLC. All rights reserved.
```

Comment:
Time travelling copyright is interesting :P Also an interesting question whether this file's copyright should be FRO if neither of the authors are associated with the FRO.

Reply:
The year is indeed wrong. Not sure what to do with the copyright: a tiny part of the original file (the declaration of the pass) remains, not sure what the copyright implications of that are